packages/appkit/src/client.ts

```diff
@@ -416,9 +417,18 @@ export class AppKit {
       sdkVersion: SdkVersion
     }
   ) {
+    OptionsController.setDebug(options.debug)
     OptionsController.setProjectId(options.projectId)
     OptionsController.setSdkVersion(options.sdkVersion)
 
+    if (!options.projectId) {
+      AlertController.open(ErrorUtil.ALERT_ERRORS.PROJECT_ID_NOT_CONFIGURED, 'error')
```

Comment:
As another solution, maybe we could map short and long messages in the `ErrorUtil` constant, and inside the `.open` method, use short message for AlertController state, and use long error for `console.error`. 

Somehow I still think we could lower the LoC and use single `console.error` and lint override, wdyt? @magiziz

Reply:
Good idea. I pushed a change where i use `shortMessage` and `longMessage` in `ErrorUtil`.